src/core/less/structure/block.less

```diff
@@ -0,0 +1,39 @@
+// Block styling
+// =============
+
+.block {
+    position: relative;
+    background-color: @background-color;
+
+    .block-inner {
+        max-width: @device-width-large;
+        margin: auto;
+        padding: @block-padding-top 0 @block-padding-bottom;
```

Comment:
it would be nice to have:
```less
@block-padding: 20px;
@block-padding-top: @block-padding;
@block-padding-right: 0;
@block-padding-bottom: @block-padding;
@block-padding-left: 0;
@block-padding-all: @block-padding-top @block-padding-right @block-padding-bottom @block-padding-left;
```

then we could replace this with ``@block-padding-all;``